packages/e2e-tests/specs/editor/plugins/align-hook.test.js

```diff
@@ -14,121 +14,137 @@ import {
 } from '@wordpress/e2e-test-utils';
 
 const alignLabels = {
+	none: 'None',
 	left: 'Align left',
 	center: 'Align center',
 	right: 'Align right',
 	wide: 'Wide width',
 	full: 'Full width',
 };
 
-describe( 'Align Hook Works As Expected', () => {
-	beforeAll( async () => {
-		await activatePlugin( 'gutenberg-test-align-hook' );
+/**
+ * Helper function to get the `labels` of align control. It actually evaluates the
+ * basic label of the button without the `info` text node if exists.
+ *
+ * @param {Object}  options                               Options for the util function
+ * @param {boolean} [options.getActiveButtonLabels=false] Flag for returning the active buttons labels only.
+ * @return {string[]} The matched labels.
+ */
+const getAlignmentToolbarLabels = async ( {
```

Comment:
I found these tests a bit hard to read so I moved the helper functions outside of  the main `describe`.